company/okrs/2020_q1.md

```diff
@@ -103,3 +102,13 @@ Build out vision for product and run customer validation discussions
 
 [N]: https://docs.google.com/document/d/1yndPaKSiB4Jq6J6cwGzcIUBSIupPOySHUHsOF2ipqFo/edit#bookmark=kix.n8t17z6iyawc
 [M]: https://docs.google.com/document/d/1yndPaKSiB4Jq6J6cwGzcIUBSIupPOySHUHsOF2ipqFo/edit#bookmark=id.vwn1af52n0ns
+
+## Operations
+
+1. **VP Business: Make Sourcegraph a lean organization that uses data and budgets to guide decision making.**
+   1. VP Business: Provide a board-approved budget (including spending and hiring plans) to each department head.
+   1. VP Business: Provide dashboards to each department head that are used on at least a weekly basis (prioritizing [sales](https://sourcegraph.looker.com/dashboards/123), [marketing](https://sourcegraph.looker.com/dashboards/118), and customer success).
+
+1. **VP Business: Build a rewarding, challenging, and inclusive people-ops operating system.**
```

Comment:
I want to formalize this responsibility, and with the hiring of a CSM (or, switching to a CE model), I'll have the ability to begin it in Q1.

Reply:
CC @mercadon check out this doc